src/test/java/io/jenkins/plugins/analysis/core/views/LocalizedPriorityTest.java

```diff
@@ -0,0 +1,146 @@
+package io.jenkins.plugins.analysis.core.views;
+
+import org.junit.jupiter.api.Test;
+
+import edu.hm.hafner.analysis.Priority;
+import static org.assertj.core.api.Assertions.*;
+
+import hudson.plugins.analysis.Messages;
+
+/**
+ * Unit test for {@link LocalizedPriority}.
+ *
+ * @author Deniz Mardin
+ * @author Frank Christian Geyer
+ */
+class LocalizedPriorityTest {
+
```

Comment:
Hier fehlt noch ein Locale.setDefault(Locale.ENGLISH); am besten in eine `@BeforeAll` Methode.